tests/framework/installer/ceph_installer.go

```diff
@@ -112,6 +113,9 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 }
 
 func (h *CephInstaller) startAdmissionController(namespace string) error {
+	if !h.k8shelper.VersionAtLeast("v1.15.0") {
```

Comment:
The 1.14 tests are actually failing because the wipe job assumes the namespace already exists. So let's move this check after the call to `CreateNamespace()`.

Reply:
done

Reply:
Please move this version check after the error checking for creating the namespace

Reply:
done